tests/commands/create_project.py

```diff
@@ -0,0 +1,269 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) Pootle contributors.
+#
+# This file is a part of the Pootle project. It is distributed under the GPL3
+# or later license. See the LICENSE file for a copy of the license and the
+# AUTHORS file for copyright and authorship information.
+
+import pytest
+
+from django.core.exceptions import ValidationError
+from django.core.management import call_command, CommandError
+
+from pootle.core.plugin import provider
+from pootle_fs.delegate import fs_plugins
+from pootle_fs.plugin import Plugin
+from pootle_project.models import Project
+
+
+@pytest.mark.cmd
+@pytest.mark.django_db
+def test_cmd_create_project_defaults(settings):
```

Comment:
Is it possible to create a project without specifying a "FS path or URL"?

Reply:
yes - as in the project form it defaults to `{POOTLE_TRANSLATION_DIRECTORY}$projectcode`